packages/components/src/components/buttons/Pin/index.tsx

```diff
@@ -1,30 +1,30 @@
 import React from 'react';
 import styled from 'styled-components';
-import { colors } from '../../../config';
 
 const Button = styled.button`
-    width: 80px;
-    height: 80px;
-    border: 1px solid ${colors.BLACK0};
-    background: ${colors.WHITE};
+    max-width: 100px;
+    max-height: 100px;
     transition: all 0.3s;
     position: relative;
     cursor: pointer;
+    margin: 4px;
+
+    padding-top: 30%;
+    width: 100%;
+
+    border-radius: 4px;
+    border: 1px solid ${props => props.theme.STROKE_GREY};
+    background: ${props => props.theme.BG_LIGHT_GREY};
```

Comment:
just ideas, haven't checked yet, but:
1. doesn't it break some nested modal pin window?
1. does it keep buttons squared?

Reply:
ohh, it should be the same. I noticed there were two pin button components. One in suite and one in trezor/components. We were using one from suite, so I copied its guts to the trezor/component's pin button and switched the import.